### PR TITLE
CONSOLE-2399: Update yarn to latest stable 1.22.10

### PR DESCRIPTION
**Pull request by zherman0**
Update yarn to latest stable 1.22.10

[CONSOLE-2399](https://issues.redhat.com/browse/CONSOLE-2399)

### CODE REVIEW

File: frontend/.yarnrc [hunk 1]
```diff
@@ -1 +1,5 @@
-yarn-path ".yarn/releases/yarn-1.21.0.js"
```
Comment (spadgett) [inline frontend/.yarnrc:1]: We should delete the old `yarn-1.21.0.js` file

**Review by spadgett — COMMENTED** [review]
Thanks @zherman0 👍

**Review by dtaylor113 — APPROVED** [review]
/lgtm

**Review by spadgett — APPROVED** [review]
/lgtm

**Comment by spadgett** [comment]
Dev-only dependency, so this shouldn't need QE review.

**Comment by zherman0** [comment]
@spadgett - Is there anything needed more for this?  I think we can update yarn without any issue.

**Pull request merged**